unit/models/codecs.py

```diff
@@ -101,6 +101,9 @@
         "recurringDebitAchPayment": lambda _id, _type, attributes, relationships:
         RecurringDebitAchPaymentDTO.from_json_api(_id, _type, attributes, relationships),
 
+        "bulkPayments": lambda _id, _type, attributes, relationships:
+        BulkPaymentsDTO.from_json_api(_id, _type, attributes, relationships),
```

Comment:
Bulk payment don't have an in top level of json. Id is inside attributes "bulkId"
<img width="758" alt="image" src="https://github.com/unit-finance/unit-python-sdk/assets/97832762/85886f04-7659-4cac-b831-911b99f3accd">